### PR TITLE
Minor typos

### PR DESCRIPTION
**Pull request by yaronf**
Minor typos

### CODE REVIEW

File: draft-ietf-gnap-core-protocol.md [hunk 1]
```diff
@@ -3881,7 +3881,7 @@ using the fictitious `photo-api` type definition.
 The access requested for a given object when using these fields 
 is the cross-product of all fields of the object. That is to 
 say, the object represents a request for all `action` values listed within the object
-to be used at all `locations` values listed within the object for all `datatype`
+to be used at all `location` values listed within the object for all `datatype`
```
Comment (jricher) [inline draft-ietf-gnap-core-protocol.md:3884]: The parameter name is plural, but we can reword the sentence so it reads better:

```suggestion
to be used at all values listed in the `locations` parameter for all `datatype`
```

Comment (yaronf) [reply draft-ietf-gnap-core-protocol.md:3884]: I was trying to be consistent with how the other parameters are named in this paragraph, i.e. plural->singular.

Comment (jricher) [reply draft-ietf-gnap-core-protocol.md:3884]: Oh good catch -- in that case, it looks like the whole paragraph needs to be rewritten to make the parameter names in the descriptions match up with the actual parameter names. I hadn't noticed the inconsistency before.

Comment (jricher) [reply draft-ietf-gnap-core-protocol.md:3884]: We'll take this change now and clean this section up with #205.

**Pull request merged**